### PR TITLE
Create corin-magee.md

### PR DESCRIPTION
**Pull request by Corin-M**

### CODE REVIEW

File: 2021/fall/Sterne-TTh@10/corin-magee.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# This is the *bio* for the cs162 session 3 prework!!
+* Who am I?
+    * I am Corin Magee. I am an M23.
+* what do you like?
+    * I like **dancing**, **laughing**, *reading*, and so much more
+
+## Random Trivia
+1. This is the first year in 10 years I haven't had dyed hair
+2. I am missing 4 non-wisdom teeth
```
Comment (Chrecci) [inline 2021/fall/Sterne-TTh@10/corin-magee.md:9]: is this all true???

**Comment by Chrecci** [comment]
Good stuff, +1

**Review by Chrecci — APPROVED** [review]
Nice work +++++1

**Comment by allisonlehn9** [comment]
Not sure if I'm supposed to tell you that you added an extra "i" in "flying." Apart from that, markdown looks good, nice!

File: 2021/fall/Sterne-TTh@10/corin-magee.md (hunk 1 above)
Comment (Corin-M) [reply 2021/fall/Sterne-TTh@10/corin-magee.md:9]: Yes it all is

**Pull request merged**